src/components/Router/index.js

```diff
@@ -31,15 +30,16 @@ class Router extends Component {
   }
 
   render = (props) => {
-    const { websocketErrorEncountered, options, ...otherProps} = props
+    const { options, ...otherProps} = props
     const defaultStepOptions = ['welcome','document','face','complete']
     const stepOptions = options.steps || defaultStepOptions
     otherProps.nextStep = this.nextStep
+    otherProps.token = options.token
+    otherProps.serverUrl = options.serverUrl
```

Comment:
this way you don't have to pass the url and token all the way to this children. do you have other ideas?